scout/cambium_updater.py

```diff
@@ -261,9 +261,9 @@ class ValidQueries(object):
     """
 
     def __init__(self):
-        self.years = ['2022', '2023']
-        self.scenarios = ['MidCase', 'Decarb95by2050',
-                          'Decarb100by2035']
+        self.years = ['2024']
+        self.scenarios = ['MidCase']
+        # , 'Decarb95by2050', 'Decarb100by2035']
```

Comment:
Ran only available case "Midcase" for Cambium updater 2024 as 'Decarb95by2050', 'Decarb100by2035' were unavailable for 2024...